### DISCUSSION TITLE
After step #15, can't get past Google Services/PIN setup

### DISCUSSION THREAD

**Discussion by luisgois** [tesla-android/issue-tracker#15] [General]
Hi,

All goes well with the Android setup until the pin creation. After confirming/retyping the pin, the Settings app crashes and we go back to enable/disable the Services (backup to drive, location, scan, updates,...).

Rpi4 8GB and 32GB micro SD card. Tried reflashing on Windows and Linux. Same outcome. How to proceed/troubleshoot this further?

Thanks

**Comment by luisgois** [comment]
I'll close it myself. Just SKIP the PIN creation (bottom left corner)

**Reply by mikegapinski** [reply]
I should consider removing the initial setup package from google apps, it makes no sense to include it

**Comment by mikegapinski** [comment]
The pin creation from setup won't work, Tesla Android does not include a Lock Screen at all
